CMakeLists.txt

```diff
@@ -80,16 +78,19 @@ if (ENABLE_OPENMP)
 endif()
 if (ENABLE_CUDA)
   list(APPEND RAJA_PERFSUITE_DEPENDS cuda)
-endif() 
-if (ENABLE_HIP)
+endif()
+# HIP is used with AMD / VEGA GPU
+# Neatly separate RAJAPerf Suite and Kokkos handling of HIP
```

Comment:
Maybe expand with a bit more detail here. IIRC it was because Kokkos requires hipcc as the CMAKE_CXX_COMPILER, whereas RAJA uses blt/CMake FindHIP